SDKTest/NeuroIDClassTests.swift

```diff
@@ -1023,7 +1022,7 @@ class NIDClientSiteIdTests: XCTestCase {
 
 class NIDSendTests: XCTestCase {
     func test_getCollectionEndpointURL() {
-        let expectedValue = "https://receiver.neuroid.cloud/c"
+        let expectedValue = "https://receiver.neuroid-dev.com/c"
```

Comment:
Looks like this test failed its assertion. The init is not setting the dev endpoint prior to running the test?

Reply:
This test passed locally for me.

Reply:
I ran the test individually. I think it has to run together since there are dependencies. Thanks! Looks like 212 tests passed. LGTM.